includes/admin/settings/class-settings-bootstrap.php

```diff
@@ -1153,23 +1153,13 @@ private static function settings_payment() {
             )
         );
 
-        wpbdp_register_setting(
-            array(
-                'id'      => 'payments-on',
-                'type'    => 'checkbox',
-                'name'    => _x( 'Turn On payments?', 'settings', 'business-directory-plugin' ),
-                'default' => false,
-                'group'   => 'payment/main',
-            )
-        );
         wpbdp_register_setting(
             array(
                 'id'           => 'payments-test-mode',
                 'type'         => 'checkbox',
                 'name'         => _x( 'Put payment gateways in test mode?', 'settings', 'business-directory-plugin' ),
                 'default'      => true,
                 'group'        => 'payment/main',
-                'requirements' => array( 'payments-on' ),
```

Comment:
It will show a blank page as there is no option to toggle payments. To show conditionally the entire payments tab will need to be hidden. Alternatively we can just leave the payments details open, like in Woo, and give the user the option. This will also by default, show new users that there are payment options available to configure, instead of hiding options from the user in the beginning

Reply:
Yep, you're right. If there isn't anything else on the page, there's no reason to hide it all.